fdbserver/workloads/TenantManagementWorkload.actor.cpp

```diff
@@ -519,6 +520,8 @@ struct TenantManagementWorkload : TestWorkload {
 							       operationType == OperationType::MANAGEMENT_DATABASE);
 							ASSERT(retried);
 							break;
+						} else if (e.code() == error_code_invalid_tenant_configuration) {
+							ASSERT_EQ(operationType, OperationType::METACLUSTER);
```

Comment:
We could also verify that the parameters were invalid (and likewise that they were valid in the success case)

Reply:
`assigned_cluster` and `ignore_capacity_limit` are randomly chosen inside `createTenantImpl`. We can add assertion for the success case there. In the `createTenant`, we do not know the value of the above two unless we return them as output arguments. Should we add output arguments to `createTenantImpl` just for the case `operationType == METACLUSTER`?

Reply:
Done